wpilibj/src/main/java/edu/wpi/first/wpilibj2/command/CommandScheduler.java

```diff
@@ -246,9 +246,9 @@ public void run() {
         for (Consumer<Command> action : m_finishActions) {
           action.accept(command);
         }
-        iterator.remove();
-
         m_requirements.keySet().removeAll(command.getRequirements());
+
```

Comment:
Remove the extra empty line, the equivalent code above doesn't have the extra whitespace, and we want them to match.